include/auth.php

```diff
@@ -4,22 +4,31 @@
 use Friendica\Core\System;
 use Friendica\Core\Config;
 use Friendica\Database\DBM;
+use Friendica\Model\User;
 
-require_once('include/security.php');
-require_once('include/datetime.php');
+require_once 'include/security.php';
+require_once 'include/datetime.php';
 
 // When the "Friendica" cookie is set, take the value to authenticate and renew the cookie.
 if (isset($_COOKIE["Friendica"])) {
 	$data = json_decode($_COOKIE["Friendica"]);
 	if (isset($data->uid)) {
-		$r = q("SELECT `user`.*, `user`.`pubkey` as `upubkey`, `user`.`prvkey` as `uprvkey`
-		FROM `user` WHERE `uid` = %d AND NOT `blocked` AND NOT `account_expired` AND NOT `account_removed` AND `verified` LIMIT 1",
-			intval($data->uid)
+
+		$user = dba::select('user',
+			[],
+			[
+				'uid' => intval($data->uid),
+				'blocked' => 0,
```

Comment:
I, personally, do prefer using "false" and "true", since it is better to read. And yes: This will work, there is some check against this: https://github.com/friendica/friendica/blob/develop/include/dba.php#L329-L331

Reply:
Ok good, I'm glad you reviewed this since it's the part I was unsure about.